app/src/main/java/android/andela/com/kenyajavadevs/util/NetworkUtility.java

```diff
@@ -0,0 +1,20 @@
+package android.andela.com.kenyajavadevs.util;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+public class NetworkUtility {
+    private Context context;
+
+    public NetworkUtility(Context context) {
+        this.context = context;
+    }
+
+    public boolean isConnectedToInternet() {
+        ConnectivityManager cm = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
```

Comment:
Good job... after resolving this two-letter variable you can merge.

Reply:
Thanks, resolved.